news/4700.feature.rst

```diff
@@ -0,0 +1,2 @@
+Change the win32 icon handler to delete the icons before assigning a new one.
+Also, using --icon=NOICON or --icon=NONE allows you to use the OS default icon.
```

Comment:
Pick one or the other, but not both. Also, the manual and command-line help text needs to be updated to reflect this new option.